gordon_janitor/plugins_loader.py

```diff
@@ -54,157 +54,4 @@
 Janitor.
 """
 
-import logging
-
-import pkg_resources
-
-from gordon_janitor import exceptions
-
-
 PLUGIN_NAMESPACE = 'gordon_janitor.plugins'
```

Comment:
Er, to clarify, you'll need to set it in the imported module, because I'm pretty sure it's only at module scope.

Reply:
from gordon import plugins_loader
plugins.loader.PLUGIN_NAMESPACE = 'gordon_janitor.plugins'

Reply:
got it!